prettier.novaextension/extension.json

```diff
@@ -343,209 +384,392 @@
-					"description": "",
+					"title": "End of Line",
+					"description": "Newline control character",
 					"link": "https://prettier.io/docs/en/options.html#end-of-line",
 					"type": "enum",
+					"radio": false,
 					"default": "Global Default",
 					"values": ["Global Default", "lf", "crlf", "cr", "auto"]
+				},
+				{
+					"key": "prettier.default-config.embeddedLanguageFormatting",
+					"title": "Embedded Language Formatting",
+					"description": "Control whether Prettier formats quoted code embedded in the file.",
+					"link": "https://prettier.io/docs/en/options.html#embedded-language-formatting",
+					"type": "enum",
+					"radio": false,
+					"default": "Global Default",
+					"values": ["Global Default", "auto", "off"]
+				},
+				{
+					"key": "prettier.default-config.singleAttributePerLine",
+					"title": "Single Attribute Per Line",
+					"description": "Enforce single attribute per line in HTML, Vue, and JSX.",
+					"link": "https://prettier.io/docs/en/options.html#single-attribute-per-line",
+					"type": "enum",
+					"radio": false,
+					"values": ["Global Default", "Enable", "Disable"],
+					"default": "Global Default"
 				}
 			]
 		},
 		{
 			"key": "prettier.format-on-save.section",
-			"title": "Format on save",
+			"title": "Format on Save Settings",
 			"type": "section",
 			"children": [
 				{
 					"key": "prettier.format-on-save",
-					"title": "Format documents when saved",
+					"title": "Format on Saved",
+					"description": "Automatically format documents when they are saved.",
 					"type": "enum",
+					"radio": false,
 					"values": ["Global Default", "Enable", "Disable"],
 					"default": "Global Default"
 				},
 				{
 					"key": "prettier.format-on-save.ignore-without-config",
-					"title": "Ignore documents without Prettier configuration file",
+					"title": "Ignore Without Config",
+					"description": "When enabled, documents without a Prettier configuration file will not be formatted on save",
 					"type": "enum",
+					"radio": false,
 					"values": ["Global Default", "Enable", "Disable"],
 					"default": "Global Default"
 				},
 				{
 					"key": "prettier.format-on-save.ignore-remote",
-					"title": "Ignore remote documents",
+					"title": "Ignore Remote Documents",
+					"description": "When enabled, remote documents will be ignored during format on save.",
 					"type": "enum",
+					"radio": false,
 					"values": ["Global Default", "Enable", "Disable"],
 					"default": "Global Default"
+				},
+				{
+					"key": "prettier.format-on-save.ignored-syntaxes.section",
+					"title": "Ignored Syntaxes",
+					"description": "Documents that are set to an ignored syntax will not be formatted when saved.",
+					"type": "section",
+					"children": [
+						{
+							"key": "prettier.format-on-save.ignored-syntaxes.css",
+							"title": "CSS",
+							"type": "enum",
+							"radio": false,
+							"values": ["Global Default", "Format on Save", "Ignore"],
```

Comment:
Nice to use the specific wording! I think it'd be nice to use [the tuple approach described in the docs](https://docs.nova.app/extensions/preferences/#enum-values) so we don't have to have the context-specific handling in `getWorkspaceConfig`

Reply:
fixed